CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased changes
 
+- Adds support for constructing "pause" and "unpause" PLT operations.
```

Comment:
Don't merge until https://github.com/Concordium/concordium-base/pull/694 is merged.

Reply:
Merged.